tests/integration/records/single_db/test_records_load_datetime.py

```diff
@@ -58,12 +59,16 @@ def load(self,
 
     def pull_result(self,
                     column_name: str) -> datetime.datetime:
-        out = self.engine.execute(f'SELECT {column_name} '
-                                  f'from {self.schema_name}.{self.table_name}')
-        ret_all = out.fetchall()
+        with self.engine.connect() as connection:
+            with connection.begin():
+                out = connection.execute(text(
+                        f'SELECT {column_name} '
+                        f'from {self.schema_name}.{self.table_name}'))
```

Comment:
I think you should be using `bindparams` or something for variable substitutions. In general using format strings to generate SQL is a big no-no because of injection vulnerabilities.

https://docs.sqlalchemy.org/en/20/core/sqlelement.html#sqlalchemy.sql.expression.TextClause.bindparams

I know the code was already like this and it's not your fault, so I'm okay with doing this as followup work but I think it's important and I don't want us to forget about it

Reply:
I created this ticket so we can go in and try and clean out all SQL injection possibilities:
https://bluelabsio.atlassian.net/browse/RM-95

Reply:
Unfortunately using bind_params or passing a dictionary of params is causing errors in the code. It seems like this isn't meant for passing table names, according to my reading. I think I'm going to investigate just using table constructs for all of these queries, like so: https://docs.sqlalchemy.org/en/20/core/selectable.html#sqlalchemy.sql.expression.table

Reply:
Going to address this in a separate ticket here @Brunope:
https://bluelabsio.atlassian.net/jira/software/projects/RM/boards/3?selectedIssue=RM-95

Reply:
Calling this resolved as we're going to address SQL Injection in the other ticket.